unizip-readme.txt

```diff
@@ -4,27 +4,24 @@ To run the game, extract the engine archive for your operating system
 into this subdirectory, so the files are right outside the pkg/ directory.
 Then, run the executable extracted from the archive.
 
-Engine archives are named like <system>-<architecture>.zip
-
-System is either, linux, windows, or macos.
-
-Architecture is either:
-
-- amd64: 64-bit x86 processor
-- i686: 32-bit x86 processor
-- arm64: 64-bit arm processor
-- armhf: 32-bit arm processor with hardware float
-
-For example windows-amd64.zip is the engine archive for running
-the game on the Windows operating system for 64-bit x86 computers.
+NOTE: In Windows Explorer using the "Extract all" function creates an unwanted extra directory,
+so it is better to put the archive's files into this directory using copy and paste.
+
+Archive for each operating system:
+	Linux, x86_64 (most common): linux-amd64.zip
+	Linux, 32-bit x86: linux-i686.zip
+	Linux, 64-bit ARM (AArch64): linux-arm64.zip
+	Linux, 32-bit ARM: linux-armhf.zip
+	Windows, 64-bit: windows-amd64.zip
+	Windows, 32-bit: windows-i686.zip
+	MacOS (either Intel or Apple Silicon): macos-amd64.zip
 
 Executable name for each operating system:
-
-- Linux: daemon
-- Windows: daemon.exe
-- macOS: Unvanquished.app
+	Linux: daemon
+	Windows: daemon.exe
+	macOS: Unvanquished.app
 
 Happy Fragging :)
 
--- 
+--
```

Comment:
The was a reference to some very old obscure RFC from the very early days of the Internet saying a mail signature would be `<minus><minus><space><return>signature`. 😂️